TixFactory.Processors.Queueing/TixFactory.Processors.Queueing/Implementation/QueueProcessor.cs

```diff
@@ -127,10 +127,9 @@ private void ProcessQueue()
 						runTask = Task.Delay(_QueueProcessorSettings.ThreadSleepTime);
 					}
 
-					_RunningTasks.Add(runTask.ContinueWith(t =>
-					{
-						_RunningTasks.Remove(runTask);
```

Comment:
This was a goof, `ContinueWith` returns a new task, not the original task. So I was trying to remove a task from the list that never existed in the list.